### ISSUE TITLE
Turkish Translation

### ISSUE THREAD

**Issue by tosbaha** [osTicket/osTicket#1799]
I have completed Turkish translation on crowdin web site.I have tried to contact authors both on osticket forum, crowdin and my efforts were fruitless. I try to create .phar file to sue with my installation. I tried to create phar file by using https://github.com/koto/phar-util First I tried to create by using the command below

```
php  include/class.translation.php messages.po > messages.po
```

However it says

```
PHP Fatal error:  Uncaught exception 'Exception' with message 'Unable to initialize MO input file' 
```

By the way is it message.po or messages.po I unpacked couple of .phar file and they all have messages.po

**Comment by greezybacon** [comment]
> php  include/class.translation.php messages.po > messages.po

This command will effectively delete your `messages.po` file. You will need to do
```
> php  include/class.translation.php messages.po > messages.mo.php
```
I'll see about getting the Turkish language pack posted.

**Comment by tosbaha** [comment]
My message had a typo sorry about that. Even though I typed php  include/class.translation.php messages.po without anything it gives the previous message.
I am using Mac Yosemite. I unpacked osticket to my desktop downloaded all files from crowdin website. I eventually end up using script file at http://osticket.com/forum/discussion/80444/add-new-language-translate-activate#latest 
Btw https://github.com/osTicket/osTicket-1.8/blob/develop/setup/doc/i18n.md file has a typo
instead of
```
php include/class.translation.php message.mo > message.mo.php
```
it should be
```
php include/class.translation.php messages.mo > messages.mo.php
```
